server/locale/DE/GPL/styles/flex/ratio--20x1.css

```
@@ -0,0 +1,64 @@
+@media (min-aspect-ratio: 200/11) {
+    .message__headline {
+        font-size: 1.8vw;
+    }
+    .message__disclaimer {
+        font-size: 1.25vw;
+        line-height: 1.43rem;
+    }
+    .message__promo-container {
+        width: unset;
+    }
+    .message__logo {
+        max-width: 128px;
+    }
+    .message__logo:nth-of-type(1) {
+        width: 18%;
+    }
+    .message__logo:nth-of-type(2) {
+        width: 60%;
+        margin: 0 4px;
+    }
+    .message__logo-container {
+        margin-right: 1.5vw;
+        max-width: 12%;
+    }
+    .message__messaging {
+        flex: unset;
+        display: inline;
+    }
+    .message__content {
+        display: flex;
+        justify-content: center;
+    }
+    .message__sub-headline {
+        margin: unset;
+    }
+    .message__logo:nth-of-type(2) {
+        display: inline-block;
+        max-width: 60%;
+    }
+    .message__headline span.weak {
+        margin-left: -4px;
+    }
+}
+@media (min-aspect-ratio: 200/11) and (max-width: 650px) {
+    .message__headline span.tag--xsmall {
+        display: inline;
+        font-size: 2.5vw;
+    }
```

Adding this makes the "mehr erfahren" text scaled a bit better relative to the rest of the message.
```suggestion
    }
    .tag--default {
        font-size: 0.6rem;
    }
```